src/modules/world/newMap/champAttacks.js

```diff
@@ -0,0 +1,53 @@
+import getElementsByTagName from '../../common/getElementsByTagName';
+import on from '../../common/on';
+import querySelectorArray from '../../common/querySelectorArray';
+
+function showChampAttack(toggle) {
+  const commonAttacks = querySelectorArray(
+    '#actionList .actionListItem.creature.creature-0 .verb.attack',
+  );
+  const uncommonAttacks = querySelectorArray(
+    '#actionList .actionListItem.creature:not(.creature-0) .verb.attack',
+  );
+
+  const [g1, g2] = toggle
+    ? [commonAttacks, uncommonAttacks]
+    : [uncommonAttacks, commonAttacks];
+
+  const blankURL = 'url("https://cdn2.fallensword.com/ui/world/icon_action_attack.png")';
+  const numberedURLPrefix = 'url("https://cdn2.fallensword.com/ui/world/icon_action_attack_';
+
+  g1.forEach((e) => { e.style.backgroundImage = blankURL; });
+  g2.filter((e, i) => i < 8).forEach((e, i) => {
+    e.style.backgroundImage = `${numberedURLPrefix}${i + 1}.png")`;
+  });
+}
+
+function champAttackListener(e) {
+  if (!e.ctrlKey
+    || !e.shiftKey
+    || e.target.tagName === 'INPUT'
+    || e.target.tagName === 'TEXTAREA') { return; }
+  showChampAttack(true);
+  if (!e.key.match(/[*&^%$#@!]/)) { return; }
```

Comment:
this looks like it would only work on a US keyboard?

Reply:
Whoops, my bias is showing! Switched to using the e.code attribute.